kong/kong.lua

```diff
@@ -1,4 +1,4 @@
--- Kong, the biggest ape in town
+-- Kong, the biggest ape in town 1
```

Comment:
true, that part you'll have to do on the tool itself, if travis, you can use ENV variables to limit it to a branch ...